main.py

```diff
@@ -324,6 +333,7 @@ def get_lldp_interfaces(self):
     def disable_lldp(self):
         """Disables an interface to receive LLDP packets."""
         interface_ids = self._get_data(request)
+        changed_interfaces = interface_ids
```

Comment:
Move this line to the line just below the modification (`interface.lldp = False`).